0.5/articles/unit-testing-elements.md

```diff
@@ -205,7 +205,7 @@ In this tutorial, we're going to add an existing Polymer element, [`<core-select
 
 First, get `<seed-element>` setup in a new `development` directory (if you haven't already in the Quick Start):
 
-    # Create a new directory to work in
+    # Create a new directory to work in (?)
```

Comment:
We assumed originally almost all devs would, but after taking some developers through the first draft of the guide it became clear that they preferred it being explicitly called out rather than an assumption. Can we remove the `(?)`?